docs/concepts/fundamentals/render_api.md

```diff
@@ -0,0 +1,121 @@
+When a component is being rendered, whether with [`Component.render()`](../../../reference/api#django_components.Component.render)
```

Comment:
This is the new section in the docs which describes what extra attributes are available during render from within methods like `get_context_data()`.